roofit/roofitcore/inc/RooMinimizer.h

```diff
@@ -132,5 +150,4 @@ class RooMinimizer : public TObject {
   ClassDef(RooMinimizer,0) // RooFit interface to ROOT::Fit::Fitter
```

Comment:
Since you added the `override` keyword to the constructor, you also have to use the alternative `ClassDef` macro that uses the override keyword. It's called `ClassDefOverride`. Like this you avoid the compiler warning about inconsistent use of override.

Reply:
Haha, yep, I was just adding this in a fix commit :) Thanks for the confirmation!

Reply:
Ah nice I was not sure if you were already on it with the fixes! I just tried to build the PR again to confirm that the warnings about not returning in a non-void function are gone, and they are :+1: